specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBManagedCassandraDataCenterGet.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "cassandra-prod-rg",
+    "api-version": "2020-06-01-preview",
+    "clusterName": "cassandra-prod",
+    "dataCenterName": "dc1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/cassandra-prod-rg/providers/Microsoft.DocumentDB/cassandraClusters/cassandra-prod/dataCenters/dc1",
+        "name": "dc1",
+        "type": "Microsoft.DocumentDB/cassandraClusters/dataCenters",
+        "properties": {
```

Comment:
The properties also contain "base64EncodedCassandraYamlFragment"

Reply:
Good catch!